libmscore/textbase.cpp

```diff
@@ -1645,6 +1689,19 @@ void TextBase::selectAll(TextCursor* _cursor)
       _cursor->setColumn(_cursor->curLine().columns());
       }
 
+//---------------------------------------------------------
+//   multiClickSelect
+//    for double and triple clicks
+//---------------------------------------------------------
+
+void TextBase::multiClickSelect(EditData& editData, int clicks)
+      {
+      if (clicks == 2)
```

Comment:
What about single click, i.e. if `clicks` is 1?

Reply:
I've changed the int parameter to an enum so the code/intent is clearer. There are only two cases - double or triple click - handled by this method and that's now fully explicit.